src/main/java/org/opensearch/sdk/handlers/ExtensionsRestRequestHandler.java

```diff
@@ -50,6 +60,15 @@ public ExtensionsRestRequestHandler(ExtensionRestPathRegistry restPathRegistry)
      */
     public RestExecuteOnExtensionResponse handleRestExecuteOnExtensionRequest(ExtensionRestRequest request) {
 
+        RestRequest.Method method = request.method();
```

Comment:
you probably don't need to extract any of these things from the request here since you are passing the request object itself into the SDKHttpRequest constructor.   You'll extract them there and assign them to those fields.